airflow/providers/microsoft/azure/provider.yaml

```
@@ -105,6 +110,9 @@ sensors:
   - integration-name: Microsoft Azure Blob Storage
     python-modules:
       - airflow.providers.microsoft.azure.sensors.wasb
+  - integration-name: Microsoft Azure Data Factory
+    python-modules:
+      - airflow.providers.microsoft.azure.sensors.azure_data_factory
```

This doesn't comply with `AIP-21: Changes in import paths`.
```suggestion
      - airflow.providers.microsoft.azure.sensors.data_factory
```

I know there are other Azure operators/sensor that aren't comply as well. We will need fix someday but lets not introduce new operators/sensor that aren't compatible with the AIP.

---

Again just following existing Azure conventions but certainly don't want to stray from an AIP. I'll update this. Thanks for the note.